src/main/frontend/components/page_menu.cljs

```diff
@@ -78,7 +78,8 @@
           file-sync-graph-uuid (and (user-handler/logged-in?)
                                     (file-sync-handler/enable-sync?)
                                     (file-sync-handler/get-current-graph-uuid))]
-      (when (and page (not block?))
+      (when (and page (not (or block?
+                               config/publishing?)))
```

Comment:
Looks like you wanted to disable favoriting but accidentally disabled the whole page menu. It's easy to fix so I can do it

Reply:
ahh, yeah I def didn't make sure I was adding that condition to the right block. 😬 thanks!!

so will this be included in the next nightly release?